truss/templates/server/common/truss_server.py

```diff
@@ -166,3 +179,78 @@ def create_application(self):
                 Exception: errors.generic_exception_handler,
             },
         )
+
+    def start(self):
+        cfg = uvicorn.Config(
+            self.create_application(),
+            host="0.0.0.0",
+            port=self.http_port,
+            workers=self.workers,
+            log_config={
+                "version": 1,
+                "formatters": {
```

Comment:
Not sure what the desired behavior you're describing is. Can you share a code snippet?

Reply:
inside `truss.templates.server.common.logging.setup_logging` there is a snippet of: 
```
        # some special handling for request logging
        if logger.name == "uvicorn.access":
            logger.addFilter(HealthCheckFilter())
```

which was to address Knative restricting visibility here.  The root of the comment here is that we should make sure we only configure logging in one place (or use static values from one place)

Reply:
I am not sure I'm following the best way to do this. We can move all these to `common/logging.py` and set things up there or we can move that filter here.

Either way, I'm not super comfortable doing that part in this PR since I don't fully understand how to do it cleanly and I think it effect logging/promtail configs downstream, so I'd prefer if you can pick it up in a quick follow up PR. Would that be okay?